Playlist.swift

```diff
@@ -6,7 +6,6 @@ public struct Playlist: Codable {
     public var name: String = ""
     public var description: String = ""
     public var ispublic: Bool = false
-    public var user_id: Int = 1
- 
+    public var userId: Int = 1
```

Comment:
It's not common to have underscores in Swift code, instead the standard is to use `camelCase`

Reply:
Yeah, I just did it in the playground to try and simplify. The playlist is a class in the "real" code and I've got CodingKeys and all the other boilerplate you need.

I do have a bunch of this kind of coding standard stuff to fix though especially naming conventions between the Postgres DB columns and Swift.